portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,17 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Brandon Vu's Portfolio</h1>
+      <p>I am a first year at UC Irvine majoring in Computer Science.
+        In my free time I like to hike, drink water and play with this japanese toy called the kendama.<p/>
```

Comment:
The kendama.jpg file that you added is never shown in this page. To keep the file structure clean, I suggest either showing it on this page somewhere, or deleting it from the images folder. Unless you're planning to show it somewhere later?